Document port mapping caveat (#811)
cirruslabs/cirrus-ci-docs: docs/guide/writing-tasks.md

??? tip "Port Mapping"
    It's also possible to map ports of additional containers by using `<HOST_PORT>:<CONTAINER_PORT>` format for the `port` field.
    For example, `port: 80:8080` will map port `8080` of the container to be available on local port `80` within a task.
  

>     **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers

Reviewer: ```suggestion
    **Note:** don't use port mapping unless absolutely necessary. A perfect use case is when you have several additional containers
```

Fixes abruptly ending sentence.

Reviewer 2: That is much better.

>     **Note:** don't use port mapping if it's not absolutly nessesary. For example, when you have several additional containers

Reviewer: ```suggestion
    **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers
```

>     which start the service on the same port and there's no easy way to change that. Port mapping limits 

Reviewer 2: Grammar - this sentence ends too abruptly

Author: Which one? "Port mapping limits"? There is continuation on the next line. 🤔

Reviewer 2: `that. Port mapping limits` You are giving an example here, and then end the sentence.  You don't explain what this example is for until the next sentence, which makes very little sense,

Reviewer: See https://github.com/cirruslabs/cirrus-ci-docs/pull/811/files#r595227933 for a fix.

>     which expose the same port and you don't have control over changing the default port number. Port mapping limits 

Reviewer: ```suggestion
    which start the service on the same port and there's no easy way to change that. Port mapping limits 
```
